.github/workflows/ci.yaml

```
@@ -27,6 +27,6 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v2
       with:
-        go-version: '1.17'
+        go-version: '1.18'
```

2 possible choices here if we don't want to maintain this

1.

```suggestion
        go-version: '>=1.18'
```

2.

```suggestion
        go-version-file: 'go.mod'
```